pkg/configuration/roles/bigchaindb/defaults/main.yml

```diff
@@ -1,31 +1,22 @@
 ---
 dependencies_deb:
   - g++
-  - python3-dev
   - libffi-dev
   - build-essential
   - libssl-dev
-  - python3-pip
```

Comment:
The instructions https://docs.bigchaindb.com/projects/server/en/master/simple-network-setup.html#install-bigchaindb-server are different.

Reply:
Removed this because, I install py36 and pip separately, in the `py36` role, under `pkg/configuration/roles/py36`